src/ambianic/pipeline/store.py

```diff
@@ -131,6 +131,71 @@ def _save_sample(self,
         self.notify(save_json)
         return image_path, json_path
 
+    def _save_sample_new(self,
```

Comment:
@ivelin 

As I am not enabled to test it on UI, doesn't know the impact on UI, here I have added sample function for fall detection inference result processing. 

Please take a look at it. Is it the correct way?

Reply:
@bhavikapanara were you able that this change does not break the UI?